build/format-notecards.js

```diff
@@ -0,0 +1,15 @@
+// Remove h4s from any existing notecards and transform them
+// from <div class="note notecard"><h4>Note:</h4>foobar</div> to
+// <div class="note notecard"><p><strong>Note:</strong>foobar</p></div>
+function formatNotecards($) {
+  $(".note.notecard h4").each((_, element) => {
```

Comment:
Shouldn't that be `div.notecard`? Because `.note` is just *one* type of `.notecard`. E.g.

```html
<div class="notecard warning">
  <h4>Warning</h4>
  ...
```
in files/en-us/web/http/headers/connection/index.html

Reply:
Yes, good catch!